scripts/peering-prefix

```diff
@@ -78,11 +85,33 @@ withdraw    Withdraw prefix to one or all muxes.
 -o asn      Prepend announcement to include the given ASN as the
             origin of the announcement.  Cannot be combined with -p;
             sets -P to 1 if not specified.  [default: unchanged (47065)]
--c id       Attach community (47065,id) to the announcement.  This
-            will let the announcement through the peer with the given
-            id.  May be added multiple times to announce to
-            multiple peers.  [default: announce to all peers]
-            (see https://peering.usc.edu/peers for the list of peers)
+
+-c id       Attach community (47065,id) to the announcement.  This will 
+            let the announcement through the peer with the given id
+            (id <= 1000) or specifically deny advertising to a peer
+            (if 1000 < id + 1000 <= 2000; e.g. to deny advertising to
+            peer 1: -c 1001).  May be added multiple times to announce
+            to multiple peers, but each peer will only see their
+            respective community (e.g.: -c 1 means Peer 1 receives
+            (47065,1)).  [default: announce to all peers only if no
+            communities attached] (see https://peering.usc.edu/peers
+            for the list of peers).  Can be combined with the -C.
+
+-C as1,c2   Attach community (id1,id2) to the announcement.  If the
+            community is in the form of (47065,id2) the effect is
+            identical to the -c option.  Similarly, if any communities
+            are in the form (47065,id2) the mux will only affect the
+            advertisement to the peer with id2 (advertise if id2 <=
+            1000, do not advertise if 1000 < id2 <= 2000) .  May be
+            added multiple times to  announce to multiple peers.
+
+            Only valid PEERING 16-bit AS numbers are allowed for id1;
+            the only exception is for the well-known communities:
```

Comment:
This isn't clear -- can I use any value for `id1` as long as `id2` is in set `(65535, 65281 - 65284)`?